packages/hash/frontend/src/blocks/page/createSuggester/createSuggester.tsx

```diff
@@ -217,7 +252,7 @@ export const createSuggester = (
         update(view) {
           const state = suggesterPluginKey.getState(view.state)!;
 
-          if (!state.isOpen()) return this.destroy!();
+          if (!view.hasFocus() || !state.isOpen()) return this.destroy!();
```

Comment:
Why is this change necessary?

Reply:
So that when we focus out of the page or comment prosemirror document the suggester closes. 
If you opened the suggester in a block and then also opened it in a comment they would both stay open.

Reply:
This made it so clicking on an item inside the suggester would close the suggester without adding the new mention/block. Had to make this fix: 8246b89a5adb7c74cfad9b9ed7c79768915ee429.

Reply:
Ah this is what I was worried about possibly happening, but you fixed it. Cool!